src/ngraph/serializer.cpp

```diff
@@ -941,14 +942,46 @@ static shared_ptr<ngraph::Function>
                 node = make_shared<op::DynPad>(args[0], args[1], args[2], args[3]);
                 break;
             }
+            case OP_TYPEID::DynReplaceSlice:
+            {
+                auto lower_bounds_mask = node_js.at("lower_bounds_mask").get<set<size_t>>();
+                auto upper_bounds_mask = node_js.at("upper_bounds_mask").get<set<size_t>>();
+                auto new_axis = node_js.at("new_axis").get<set<size_t>>();
+                auto shrink_axis = node_js.at("shrink_axis").get<set<size_t>>();
+                auto ellipsis_mask = node_js.at("ellipsis_mask").get<set<size_t>>();
+                node = make_shared<op::DynReplaceSlice>(args[0],
+                                                        args[1],
+                                                        args[2],
+                                                        args[3],
+                                                        args[4],
+                                                        lower_bounds_mask,
+                                                        upper_bounds_mask,
+                                                        new_axis,
+                                                        shrink_axis,
+                                                        ellipsis_mask);
+                break;
+            }
             case OP_TYPEID::DynReshape:
             {
                 node = make_shared<op::DynReshape>(args[0], args[1]);
                 break;
             }
             case OP_TYPEID::DynSlice:
             {
-                node = make_shared<op::DynSlice>(args[0], args[1], args[2], args[3]);
+                auto lower_bounds_mask = node_js.at("lower_bounds_mask").get<set<size_t>>();
```

Comment:
attn @rkimballn1: While I was here, I noticed `DynSlice` was not serializing its attributes. NB, the update here is not backwards compatible. Since this is an `experimental` op I think that's probably okay, but let me know if you want me to fix that.